librz/core/cmd/cmd_analysis.c

```diff
@@ -5544,7 +5544,7 @@ static void core_colorify_il_statement(RzConsContext *ctx, const char *il_stmt,
 	rz_cons_newline();
 }
 
-static void core_analysis_bytes_il(RzCore *core, const ut8 *buf, int len, int nops, bool pretty) {
+RZ_API void rz_core_analysis_bytes_il(RzCore *core, const ut8 *buf, int len, int nops, bool pretty) {
```

Comment:
Please add Doxygen then

Reply:
And add `rz_return_if_fail(core && buf)`

Reply:
add also RZ_NULLABLE/NONNULL etc..